INFO 2025-09-02 11:15:23,230 views 69169 6204518400 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-02 11:15:23,230 views 69169 6204518400 Processing authentication with request_token: FLYVzcnPjd...
INFO 2025-09-02 11:15:23,230 views 69169 6204518400 Attempting to generate session...
ERROR 2025-09-02 11:15:23,729 services 69169 6204518400 Session generation failed: Token is invalid or has expired.
ERROR 2025-09-02 11:15:23,730 views 69169 6204518400 Authentication failed: Token is invalid or has expired.
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 51, in settings
    session_data = kite_service.generate_session(request_token)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 75, in generate_session
    session_data = self.kite.generate_session(
        request_token, api_secret=self.api_secret
    )
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 263, in generate_session
    resp = self._post("api.token", params={
        "api_key": self.api_key,
        "request_token": request_token,
        "checksum": checksum
    })
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 865, in _post
    return self._request(route, "POST", url_args=url_args, params=params, is_json=is_json, query_params=query_params)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 937, in _request
    raise exp(data["message"], code=r.status_code)
kiteconnect.exceptions.TokenException: Token is invalid or has expired.
INFO 2025-09-02 11:15:29,057 views 69169 6204518400 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-02 11:15:29,057 views 69169 6204518400 Processing authentication with request_token: FLYVzcnPjd...
INFO 2025-09-02 11:15:29,057 views 69169 6204518400 Attempting to generate session...
ERROR 2025-09-02 11:15:29,388 services 69169 6204518400 Session generation failed: Token is invalid or has expired.
ERROR 2025-09-02 11:15:29,389 views 69169 6204518400 Authentication failed: Token is invalid or has expired.
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 51, in settings
    session_data = kite_service.generate_session(request_token)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 75, in generate_session
    session_data = self.kite.generate_session(
        request_token, api_secret=self.api_secret
    )
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 263, in generate_session
    resp = self._post("api.token", params={
        "api_key": self.api_key,
        "request_token": request_token,
        "checksum": checksum
    })
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 865, in _post
    return self._request(route, "POST", url_args=url_args, params=params, is_json=is_json, query_params=query_params)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 937, in _request
    raise exp(data["message"], code=r.status_code)
kiteconnect.exceptions.TokenException: Token is invalid or has expired.
INFO 2025-09-02 11:17:47,173 views 69169 6204518400 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-02 11:17:47,173 views 69169 6204518400 Processing authentication with request_token: n8ItarV61y...
INFO 2025-09-02 11:17:47,173 views 69169 6204518400 Attempting to generate session...
INFO 2025-09-02 11:17:47,605 views 69169 6204518400 Session data received: {'user_type': 'individual/res_no_nn', 'email': 'bhatikares@gmail.com', 'user_name': 'Sourabh Vidhyadhar Bhatikare', 'user_shortname': 'Sourabh', 'broker': 'ZERODHA', 'exchanges': ['NSE', 'NFO', 'BFO', 'BSE', 'MF'], 'products': ['CNC', 'NRML', 'MIS', 'BO', 'CO'], 'order_types': ['MARKET', 'LIMIT', 'SL', 'SL-M'], 'avatar_url': None, 'user_id': 'WH8704', 'api_key': 'bd53m7sxfmjy2kbz', 'access_token': 'OP125jPtFBmRR29bdoZsMDcuTy3J6v5Q', 'public_token': '7SbmT9NeoOGqhrTNR3pxQQwFsggGNVGR', 'refresh_token': '', 'enctoken': 'Ux4Z3FMuYL5hbwTRsUZ0b/actaBi6ONNXnZS7PHync7FNYH4bxv3xxuTT+398GApKLLlWYVUa+meGJ8VO+DoMVHz+fLmcFOaMQVer2xtDgDopbZXJ2usyeAYd9s7yrY=', 'login_time': datetime.datetime(2025, 9, 2, 16, 34, 37), 'meta': {'demat_consent': 'consent'}}
INFO 2025-09-02 11:17:47,609 views 69169 6204518400 Authentication successful for user: WH8704
ERROR 2025-09-02 11:18:11,675 views 69169 6204518400 Error fetching data: KiteDataService.fetch_historical_data() got an unexpected keyword argument 'symbol'
INFO 2025-09-02 11:20:46,874 services 69281 6165934080 Fetching data for token 2953217 from 2025-08-26 to 2025-09-02
INFO 2025-09-02 11:21:13,988 services 69281 6165934080 Fetching data for token 341249 from 2025-08-26 to 2025-09-02
ERROR 2025-09-02 11:32:45,526 views 69403 6160379904 Error fetching data: KiteDataService.load_data_from_json() takes 2 positional arguments but 5 were given
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 41, in index
    existing_data = kite_service.load_data_from_json(symbol, from_date, to_date, interval)
TypeError: KiteDataService.load_data_from_json() takes 2 positional arguments but 5 were given
INFO 2025-09-02 11:34:05,090 services 69576 6205370368 Fetching data for token 738561 from 2025-08-03 to 2025-09-02
ERROR 2025-09-02 11:34:05,728 services 69576 6205370368 Error saving data to JSON: 'str' object has no attribute 'strftime'
ERROR 2025-09-02 11:34:05,728 views 69576 6205370368 Error fetching data: 'str' object has no attribute 'strftime'
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 55, in index
    filepath = kite_service.save_data_to_json(historical_data, symbol, from_date, to_date, interval)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 407, in save_data_to_json
    from_date_str = from_date.strftime("%Y%m%d")
                    ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'strftime'
INFO 2025-09-02 11:34:09,641 services 69576 6205370368 Fetching data for token 408065 from 2025-08-03 to 2025-09-02
ERROR 2025-09-02 11:34:10,382 services 69576 6205370368 Error saving data to JSON: 'str' object has no attribute 'strftime'
ERROR 2025-09-02 11:34:10,382 views 69576 6205370368 Error fetching data: 'str' object has no attribute 'strftime'
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 55, in index
    filepath = kite_service.save_data_to_json(historical_data, symbol, from_date, to_date, interval)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 407, in save_data_to_json
    from_date_str = from_date.strftime("%Y%m%d")
                    ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'strftime'
INFO 2025-09-02 11:36:06,140 services 69610 6194507776 Fetching data for token 408065 from 2025-08-03 to 2025-09-02
ERROR 2025-09-02 11:36:06,705 services 69610 6194507776 Error saving data to JSON: 'str' object has no attribute 'strftime'
ERROR 2025-09-02 11:36:06,705 views 69610 6194507776 Error fetching data: 'str' object has no attribute 'strftime'
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 69, in index
    filepath = kite_service.save_data_to_json(historical_data, symbol, from_date, to_date, interval)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 407, in save_data_to_json
    from_date_str = from_date.strftime("%Y%m%d")
                    ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'strftime'
INFO 2025-09-02 11:36:10,072 services 69610 6194507776 Fetching data for token 408065 from 2025-08-03 to 2025-09-02
ERROR 2025-09-02 11:36:10,667 services 69610 6194507776 Error saving data to JSON: 'str' object has no attribute 'strftime'
ERROR 2025-09-02 11:36:10,667 views 69610 6194507776 Error fetching data: 'str' object has no attribute 'strftime'
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 69, in index
    filepath = kite_service.save_data_to_json(historical_data, symbol, from_date, to_date, interval)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 407, in save_data_to_json
    from_date_str = from_date.strftime("%Y%m%d")
                    ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'strftime'
INFO 2025-09-02 11:38:23,156 services 69659 6202798080 Fetching data for token 408065 from 2025-08-03 to 2025-09-02
INFO 2025-09-02 11:38:23,717 services 69659 6202798080 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/INFY_minute_20250803_to_20250902_20250902_113823.json
INFO 2025-09-02 11:38:38,715 services 69659 6202798080 Fetching data for token 408065 from 2025-08-03 to 2025-09-02
INFO 2025-09-02 11:38:39,329 services 69659 6202798080 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/INFY_minute_20250803_to_20250902_20250902_113839.json
INFO 2025-09-02 11:39:46,515 services 69872 6166065152 Fetching data for token 738561 from 2025-08-03 to 2025-09-02
INFO 2025-09-02 11:39:47,128 services 69872 6166065152 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/RELIANCE_minute_20250803_to_20250902_20250902_113947.json
INFO 2025-09-02 11:40:04,957 services 69872 6166065152 Fetching data for token 2939649 from 2025-08-01 to 2025-09-02
INFO 2025-09-02 11:40:05,495 services 69872 6166065152 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/LT_minute_20250801_to_20250902_20250902_114005.json
INFO 2025-09-02 15:26:58,184 services 71680 6197096448 Fetching data for token 2953217 from 2025-08-03 to 2025-09-02
INFO 2025-09-02 15:26:58,880 services 71680 6197096448 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/TCS_minute_20250803_to_20250902_20250902_152658.json
INFO 2025-09-02 17:45:40,451 strategy_service 74297 12901707776 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/TCS_minute_20250803_to_20250902_20250902_152658.json
INFO 2025-09-02 17:45:40,603 strategy_service 74297 12901707776 Saved 20 signals for TCS
ERROR 2025-09-02 17:45:40,604 strategy_service 74297 12901707776 Error saving backtest results: keys must be str, int, float, bool or None, not Timestamp
ERROR 2025-09-02 17:45:40,604 strategy_service 74297 12901707776 Error running strategy: keys must be str, int, float, bool or None, not Timestamp
INFO 2025-09-02 17:45:47,366 strategy_service 74297 12901707776 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/TCS_minute_20250803_to_20250902_20250902_152658.json
INFO 2025-09-02 17:45:47,501 strategy_service 74297 12901707776 Saved 20 signals for TCS
ERROR 2025-09-02 17:45:47,502 strategy_service 74297 12901707776 Error saving backtest results: keys must be str, int, float, bool or None, not Timestamp
ERROR 2025-09-02 17:45:47,502 strategy_service 74297 12901707776 Error running strategy: keys must be str, int, float, bool or None, not Timestamp
INFO 2025-09-02 17:47:01,100 strategy_service 74297 12901707776 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/INFY_minute_20250803_to_20250902_20250902_113839.json
INFO 2025-09-02 17:47:01,250 strategy_service 74297 12901707776 Saved 21 signals for INFY
ERROR 2025-09-02 17:47:01,250 strategy_service 74297 12901707776 Error saving backtest results: keys must be str, int, float, bool or None, not Timestamp
ERROR 2025-09-02 17:47:01,250 strategy_service 74297 12901707776 Error running strategy: keys must be str, int, float, bool or None, not Timestamp
INFO 2025-09-02 17:48:44,683 strategy_service 74297 12901707776 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/INFY_minute_20250803_to_20250902_20250902_113839.json
INFO 2025-09-02 17:48:44,831 strategy_service 74297 12901707776 Saved 21 signals for INFY
ERROR 2025-09-02 17:48:44,831 strategy_service 74297 12901707776 Error saving backtest results: keys must be str, int, float, bool or None, not Timestamp
ERROR 2025-09-02 17:48:44,831 strategy_service 74297 12901707776 Error running strategy: keys must be str, int, float, bool or None, not Timestamp
INFO 2025-09-02 17:50:50,082 strategy_service 74511 6145126400 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/RELIANCE_minute_20250803_to_20250902_20250902_113947.json
INFO 2025-09-02 17:50:50,227 strategy_service 74511 6145126400 Saved 13 signals for RELIANCE
INFO 2025-09-02 17:50:50,228 strategy_service 74511 6145126400 Saved backtest results for RELIANCE
INFO 2025-09-02 17:58:32,745 strategy_service 74511 6145126400 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/INFY_minute_20250803_to_20250902_20250902_113839.json
INFO 2025-09-02 17:58:32,873 strategy_service 74511 6145126400 Saved 21 signals for INFY
INFO 2025-09-02 17:58:32,873 strategy_service 74511 6145126400 Saved backtest results for INFY
INFO 2025-09-02 18:08:09,118 services 74787 6133968896 Fetching data for token 2939649 from 2025-08-03 to 2025-09-02
INFO 2025-09-02 18:08:09,731 services 74787 6133968896 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/LT_minute_20250803_to_20250902_20250902_180809.json
INFO 2025-09-02 18:08:50,954 strategy_service 74787 6133968896 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/LT_minute_20250803_to_20250902_20250902_180809.json
INFO 2025-09-02 18:08:51,104 strategy_service 74787 6133968896 Saved 14 signals for LT
INFO 2025-09-02 18:08:51,105 strategy_service 74787 6133968896 Saved backtest results for LT
INFO 2025-09-02 18:29:05,002 views 74787 6133968896 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-02 18:29:05,003 views 74787 6133968896 Processing authentication with request_token: n8ItarV61y...
INFO 2025-09-02 18:29:05,003 views 74787 6133968896 Attempting to generate session...
ERROR 2025-09-02 18:29:05,260 services 74787 6133968896 Session generation failed: Token is invalid or has expired.
ERROR 2025-09-02 18:29:05,261 views 74787 6133968896 Authentication failed: Token is invalid or has expired.
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 127, in settings
    session_data = kite_service.generate_session(request_token)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 162, in generate_session
    session_data = self.kite.generate_session(
        request_token, api_secret=self.api_secret
    )
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 263, in generate_session
    resp = self._post("api.token", params={
        "api_key": self.api_key,
        "request_token": request_token,
        "checksum": checksum
    })
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 865, in _post
    return self._request(route, "POST", url_args=url_args, params=params, is_json=is_json, query_params=query_params)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 937, in _request
    raise exp(data["message"], code=r.status_code)
kiteconnect.exceptions.TokenException: Token is invalid or has expired.
INFO 2025-09-02 18:29:17,356 views 74787 6133968896 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-02 18:29:17,356 views 74787 6133968896 Processing authentication with request_token: n8ItarV61y...
INFO 2025-09-02 18:29:17,356 views 74787 6133968896 Attempting to generate session...
ERROR 2025-09-02 18:29:17,591 services 74787 6133968896 Session generation failed: Token is invalid or has expired.
ERROR 2025-09-02 18:29:17,592 views 74787 6133968896 Authentication failed: Token is invalid or has expired.
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 127, in settings
    session_data = kite_service.generate_session(request_token)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 162, in generate_session
    session_data = self.kite.generate_session(
        request_token, api_secret=self.api_secret
    )
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 263, in generate_session
    resp = self._post("api.token", params={
        "api_key": self.api_key,
        "request_token": request_token,
        "checksum": checksum
    })
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 865, in _post
    return self._request(route, "POST", url_args=url_args, params=params, is_json=is_json, query_params=query_params)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 937, in _request
    raise exp(data["message"], code=r.status_code)
kiteconnect.exceptions.TokenException: Token is invalid or has expired.
INFO 2025-09-03 08:20:52,355 strategy_service 79358 6191525888 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/TCS_minute_20250803_to_20250902_20250902_152658.json
INFO 2025-09-03 08:20:52,502 strategy_service 79358 6191525888 Saved 20 signals for TCS
INFO 2025-09-03 08:20:52,503 strategy_service 79358 6191525888 Saved backtest results for TCS
INFO 2025-09-03 12:48:27,670 views 82181 6197866496 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-03 12:48:27,671 views 82181 6197866496 Processing authentication with request_token: 9ZqnJlL4Wq...
INFO 2025-09-03 12:48:27,671 views 82181 6197866496 Attempting to generate session...
ERROR 2025-09-03 12:48:28,230 services 82181 6197866496 Session generation failed: Token is invalid or has expired.
ERROR 2025-09-03 12:48:28,230 views 82181 6197866496 Authentication failed: Token is invalid or has expired.
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 127, in settings
    session_data = kite_service.generate_session(request_token)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 162, in generate_session
    session_data = self.kite.generate_session(
        request_token, api_secret=self.api_secret
    )
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 263, in generate_session
    resp = self._post("api.token", params={
        "api_key": self.api_key,
        "request_token": request_token,
        "checksum": checksum
    })
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 865, in _post
    return self._request(route, "POST", url_args=url_args, params=params, is_json=is_json, query_params=query_params)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 937, in _request
    raise exp(data["message"], code=r.status_code)
kiteconnect.exceptions.TokenException: Token is invalid or has expired.
INFO 2025-09-03 12:48:38,122 views 82181 6197866496 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-03 12:48:38,123 views 82181 6197866496 Processing authentication with request_token: 9ZqnJlL4Wq...
INFO 2025-09-03 12:48:38,123 views 82181 6197866496 Attempting to generate session...
ERROR 2025-09-03 12:48:38,688 services 82181 6197866496 Session generation failed: Token is invalid or has expired.
ERROR 2025-09-03 12:48:38,688 views 82181 6197866496 Authentication failed: Token is invalid or has expired.
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 127, in settings
    session_data = kite_service.generate_session(request_token)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 162, in generate_session
    session_data = self.kite.generate_session(
        request_token, api_secret=self.api_secret
    )
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 263, in generate_session
    resp = self._post("api.token", params={
        "api_key": self.api_key,
        "request_token": request_token,
        "checksum": checksum
    })
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 865, in _post
    return self._request(route, "POST", url_args=url_args, params=params, is_json=is_json, query_params=query_params)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 937, in _request
    raise exp(data["message"], code=r.status_code)
kiteconnect.exceptions.TokenException: Token is invalid or has expired.
INFO 2025-09-03 12:49:06,988 views 82181 6197866496 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-03 12:49:06,988 views 82181 6197866496 Processing authentication with request_token: T9ZqnJlL4W...
INFO 2025-09-03 12:49:06,988 views 82181 6197866496 Attempting to generate session...
ERROR 2025-09-03 12:49:07,404 services 82181 6197866496 Session generation failed: Invalid `checksum`.
ERROR 2025-09-03 12:49:07,405 views 82181 6197866496 Authentication failed: Invalid `checksum`.
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 127, in settings
    session_data = kite_service.generate_session(request_token)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 162, in generate_session
    session_data = self.kite.generate_session(
        request_token, api_secret=self.api_secret
    )
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 263, in generate_session
    resp = self._post("api.token", params={
        "api_key": self.api_key,
        "request_token": request_token,
        "checksum": checksum
    })
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 865, in _post
    return self._request(route, "POST", url_args=url_args, params=params, is_json=is_json, query_params=query_params)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 937, in _request
    raise exp(data["message"], code=r.status_code)
kiteconnect.exceptions.TokenException: Invalid `checksum`.
INFO 2025-09-03 12:49:16,603 views 82181 6197866496 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-03 12:49:16,603 views 82181 6197866496 Processing authentication with request_token: T9ZqnJlL4W...
INFO 2025-09-03 12:49:16,603 views 82181 6197866496 Attempting to generate session...
ERROR 2025-09-03 12:49:17,258 services 82181 6197866496 Session generation failed: Invalid `checksum`.
ERROR 2025-09-03 12:49:17,258 views 82181 6197866496 Authentication failed: Invalid `checksum`.
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 127, in settings
    session_data = kite_service.generate_session(request_token)
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 162, in generate_session
    session_data = self.kite.generate_session(
        request_token, api_secret=self.api_secret
    )
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 263, in generate_session
    resp = self._post("api.token", params={
        "api_key": self.api_key,
        "request_token": request_token,
        "checksum": checksum
    })
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 865, in _post
    return self._request(route, "POST", url_args=url_args, params=params, is_json=is_json, query_params=query_params)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/kiteconnect/connect.py", line 937, in _request
    raise exp(data["message"], code=r.status_code)
kiteconnect.exceptions.TokenException: Invalid `checksum`.
INFO 2025-09-03 12:51:16,290 views 82181 6197866496 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-03 12:51:16,290 views 82181 6197866496 Processing authentication with request_token: 6ImNiRVIhQ...
INFO 2025-09-03 12:51:16,291 views 82181 6197866496 Attempting to generate session...
INFO 2025-09-03 12:51:16,637 views 82181 6197866496 Session data received: {'user_type': 'individual/res_no_nn', 'email': 'bhatikares@gmail.com', 'user_name': 'Sourabh Vidhyadhar Bhatikare', 'user_shortname': 'Sourabh', 'broker': 'ZERODHA', 'exchanges': ['BFO', 'BSE', 'NSE', 'MF', 'NFO'], 'products': ['CNC', 'NRML', 'MIS', 'BO', 'CO'], 'order_types': ['MARKET', 'LIMIT', 'SL', 'SL-M'], 'avatar_url': None, 'user_id': 'WH8704', 'api_key': 'bd53m7sxfmjy2kbz', 'access_token': 'GyIT5cppuFHtrjLdN3IbieJzhzlL36ka', 'public_token': 'N0h72qhWwudx9f6VXwt6JFYiGdrhNbmn', 'refresh_token': '', 'enctoken': 'Oq8uxmuEvxip64OOTcVd2u89vDSPVGaHXgtGgAMXjzlyH+BXxWLt3N0NyM7belC5NEDJ89sTGx+41X0M/emwAzoTmFugmEvBylhwf8SJo80l84SkYFRQMljbVdZgkDk=', 'login_time': datetime.datetime(2025, 9, 3, 18, 18, 11), 'meta': {'demat_consent': 'consent'}}
INFO 2025-09-03 12:51:16,639 views 82181 6197866496 Authentication successful for user: WH8704
INFO 2025-09-03 12:52:57,717 services 82181 6197866496 Fetching data for token 2815745 from 2025-08-28 to 2025-09-03
INFO 2025-09-03 12:52:58,285 services 82181 6197866496 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/MARUTI_minute_20250828_to_20250903_20250903_125258.json
INFO 2025-09-03 12:53:20,893 strategy_service 82181 6197866496 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/MARUTI_minute_20250828_to_20250903_20250903_125258.json
INFO 2025-09-03 12:53:20,978 strategy_service 82181 6197866496 Saved 9 signals for MARUTI
INFO 2025-09-03 12:53:20,979 strategy_service 82181 6197866496 Saved backtest results for MARUTI
INFO 2025-09-03 12:59:22,863 services 82181 6197866496 Fetching data for token 3787777 from 2025-08-04 to 2025-09-03
ERROR 2025-09-03 12:59:23,359 services 82181 6197866496 Error fetching historical data: invalid token
INFO 2025-09-03 12:59:23,366 services 82181 6197866496 Generated 1000 sample records
INFO 2025-09-03 12:59:23,388 services 82181 6197866496 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/WIPRO_minute_20250804_to_20250903_20250903_125923.json
INFO 2025-09-03 12:59:40,609 strategy_service 82181 6197866496 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/WIPRO_minute_20250804_to_20250903_20250903_125923.json
INFO 2025-09-03 12:59:40,665 strategy_service 82181 6197866496 Saved 5 signals for WIPRO
INFO 2025-09-03 12:59:40,666 strategy_service 82181 6197866496 Saved backtest results for WIPRO
INFO 2025-09-03 13:45:45,278 services 82181 6197866496 Fetching data for token 2953217 from 2025-08-04 to 2025-09-03
INFO 2025-09-03 13:45:47,161 services 82181 6197866496 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/TCS_minute_20250804_to_20250903_20250903_134547.json
INFO 2025-09-03 13:45:58,877 strategy_service 82181 6197866496 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/TCS_minute_20250804_to_20250903_20250903_134547.json
INFO 2025-09-03 13:45:59,034 strategy_service 82181 6197866496 Saved 20 signals for TCS
INFO 2025-09-03 13:45:59,035 strategy_service 82181 6197866496 Saved backtest results for TCS
INFO 2025-09-03 14:24:58,507 strategy_service 82181 6197866496 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/TCS_minute_20250804_to_20250903_20250903_134547.json
INFO 2025-09-03 14:24:58,659 strategy_service 82181 6197866496 Saved 20 signals for TCS
INFO 2025-09-03 14:24:58,660 strategy_service 82181 6197866496 Saved backtest results for TCS
INFO 2025-09-03 14:29:38,658 services 85009 6129594368 Fetching data for token 738561 from 2025-08-04 to 2025-09-03
INFO 2025-09-03 14:29:39,352 services 85009 6129594368 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/RELIANCE_minute_20250804_to_20250903_20250903_142939.json
INFO 2025-09-03 14:29:51,437 strategy_service 85009 6129594368 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/RELIANCE_minute_20250804_to_20250903_20250903_142939.json
INFO 2025-09-03 14:29:51,593 strategy_service 85009 6129594368 Saved 21 signals for RELIANCE
INFO 2025-09-03 14:29:51,594 strategy_service 85009 6129594368 Saved backtest results for RELIANCE
INFO 2025-09-03 14:31:07,860 views 85096 6136934400 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/RELIANCE_minute_20250804_to_20250903_20250903_142939.json
INFO 2025-09-03 14:31:07,865 views 85096 6136934400 Loaded file with 7875 records
INFO 2025-09-03 14:31:49,864 views 85115 6158626816 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/RELIANCE_minute_20250804_to_20250903_20250903_142939.json
INFO 2025-09-03 14:31:49,869 views 85115 6158626816 Loaded file with 7875 records
INFO 2025-09-03 14:31:49,871 views 85115 6158626816 Extracted 7875 valid OHLC records
INFO 2025-09-03 14:32:29,467 views 85115 6158626816 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/TCS_minute_20250804_to_20250903_20250903_134547.json
INFO 2025-09-03 14:32:29,472 views 85115 6158626816 Loaded file with 7875 records
INFO 2025-09-03 14:32:29,474 views 85115 6158626816 Extracted 7875 valid OHLC records
INFO 2025-09-03 14:32:36,822 views 85115 6158626816 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/RELIANCE_minute_20250804_to_20250903_20250903_142939.json
INFO 2025-09-03 14:32:36,827 views 85115 6158626816 Loaded file with 7875 records
INFO 2025-09-03 14:32:36,830 views 85115 6158626816 Extracted 7875 valid OHLC records
INFO 2025-09-03 14:33:22,481 views 85115 6158626816 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/TCS_minute_20250804_to_20250903_20250903_134547.json
INFO 2025-09-03 14:33:22,487 views 85115 6158626816 Loaded file with 7875 records
INFO 2025-09-03 14:33:22,489 views 85115 6158626816 Extracted 7875 valid OHLC records
INFO 2025-09-03 14:33:41,310 views 85115 6158626816 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/RELIANCE_minute_20250804_to_20250903_20250903_142939.json
INFO 2025-09-03 14:33:41,315 views 85115 6158626816 Loaded file with 7875 records
INFO 2025-09-03 14:33:41,317 views 85115 6158626816 Extracted 7875 valid OHLC records
INFO 2025-09-03 15:14:22,904 services 86096 6160478208 Fetching data for token 348929 from 2025-08-04 to 2025-09-03
INFO 2025-09-03 15:14:23,549 services 86096 6160478208 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/HINDALCO_minute_20250804_to_20250903_20250903_151423.json
INFO 2025-09-03 15:14:56,343 strategy_service 86096 6160478208 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/HINDALCO_minute_20250804_to_20250903_20250903_151423.json
INFO 2025-09-03 15:14:56,503 strategy_service 86096 6160478208 Saved 23 signals for HINDALCO
INFO 2025-09-03 15:14:56,504 strategy_service 86096 6160478208 Saved backtest results for HINDALCO
INFO 2025-09-03 15:15:46,352 views 86096 6160478208 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/HINDALCO_minute_20250804_to_20250903_20250903_151423.json
INFO 2025-09-03 15:15:46,356 views 86096 6160478208 Loaded file with 7875 records
INFO 2025-09-03 15:15:46,359 views 86096 6160478208 Extracted 7875 valid OHLC records
INFO 2025-09-03 15:16:09,033 views 86096 6160478208 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/HINDALCO_minute_20250804_to_20250903_20250903_151423.json
INFO 2025-09-03 15:16:09,038 views 86096 6160478208 Loaded file with 7875 records
INFO 2025-09-03 15:16:09,040 views 86096 6160478208 Extracted 7875 valid OHLC records
INFO 2025-09-04 17:40:03,282 views 96337 6161821696 Auth form submission - Form valid: True, Credentials exist: True
INFO 2025-09-04 17:40:03,284 views 96337 6161821696 Processing authentication with request_token: kmS2IamkGr...
INFO 2025-09-04 17:40:03,284 views 96337 6161821696 Attempting to generate session...
INFO 2025-09-04 17:40:03,392 views 96337 6161821696 Session data received: {'user_type': 'individual/res_no_nn', 'email': 'bhatikares@gmail.com', 'user_name': 'Sourabh Vidhyadhar Bhatikare', 'user_shortname': 'Sourabh', 'broker': 'ZERODHA', 'exchanges': ['BFO', 'BSE', 'NSE', 'MF', 'NFO'], 'products': ['CNC', 'NRML', 'MIS', 'BO', 'CO'], 'order_types': ['MARKET', 'LIMIT', 'SL', 'SL-M'], 'avatar_url': None, 'user_id': 'WH8704', 'api_key': 'bd53m7sxfmjy2kbz', 'access_token': 'jwUO7w7E3WTd4xEu3g9zkba2E0D03GMi', 'public_token': 'Bz7fzP8ZH3J5CLOs4JfFOSplxA65x7EZ', 'refresh_token': '', 'enctoken': 'G6PWVqqd0pQYw9ke9XIsNEcX1/YoYMAZeSxk0B/PA2rYkG2b0OSQHDL7G1HCVEYzlwFPvyzHacOKvttTVMsawP/R11JuYuGjH94u81X+wgqDSOC2vYD5Z+ls4wW0rvA=', 'login_time': datetime.datetime(2025, 9, 4, 23, 9, 38), 'meta': {'demat_consent': 'consent'}}
INFO 2025-09-04 17:40:03,395 views 96337 6161821696 Authentication successful for user: WH8704
ERROR 2025-09-04 17:48:32,282 views 96780 6133067776 Error fetching data: Symbol NIFTY 50 not found in supported stocks
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 62, in index
    historical_data = kite_service.fetch_historical_data_by_symbol(
        symbol=symbol,
    ...<2 lines>...
        interval=interval
    )
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 292, in fetch_historical_data_by_symbol
    raise ValueError(f"Symbol {symbol} not found in supported stocks")
ValueError: Symbol NIFTY 50 not found in supported stocks
ERROR 2025-09-04 17:48:41,463 views 96780 6133067776 Error fetching data: Symbol GOLDBEES not found in supported stocks
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 62, in index
    historical_data = kite_service.fetch_historical_data_by_symbol(
        symbol=symbol,
    ...<2 lines>...
        interval=interval
    )
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 292, in fetch_historical_data_by_symbol
    raise ValueError(f"Symbol {symbol} not found in supported stocks")
ValueError: Symbol GOLDBEES not found in supported stocks
INFO 2025-09-04 17:48:50,295 services 96780 6133067776 Fetching data for token 408065 from 2025-08-05 to 2025-09-04
INFO 2025-09-04 17:48:50,665 services 96780 6133067776 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/INFY_minute_20250805_to_20250904_20250904_174850.json
ERROR 2025-09-04 17:52:25,988 views 96896 6169325568 Error fetching data: Symbol GOLDBEES not found in supported stocks
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 62, in index
    historical_data = kite_service.fetch_historical_data_by_symbol(
        symbol=symbol,
    ...<2 lines>...
        interval=interval
    )
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 292, in fetch_historical_data_by_symbol
    raise ValueError(f"Symbol {symbol} not found in supported stocks")
ValueError: Symbol GOLDBEES not found in supported stocks
ERROR 2025-09-04 17:52:27,737 views 96896 6169325568 Error fetching data: Symbol GOLDBEES not found in supported stocks
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 62, in index
    historical_data = kite_service.fetch_historical_data_by_symbol(
        symbol=symbol,
    ...<2 lines>...
        interval=interval
    )
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 292, in fetch_historical_data_by_symbol
    raise ValueError(f"Symbol {symbol} not found in supported stocks")
ValueError: Symbol GOLDBEES not found in supported stocks
ERROR 2025-09-04 17:52:37,304 views 96896 6169325568 Error fetching data: Symbol NIFTY50 not found in supported stocks
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 62, in index
    historical_data = kite_service.fetch_historical_data_by_symbol(
        symbol=symbol,
    ...<2 lines>...
        interval=interval
    )
  File "/Users/prathameshkale/Documents/django-automation/stock_data/services.py", line 292, in fetch_historical_data_by_symbol
    raise ValueError(f"Symbol {symbol} not found in supported stocks")
ValueError: Symbol NIFTY50 not found in supported stocks
INFO 2025-09-04 17:52:51,786 services 96922 6133706752 Fetching data for token 4268801 from 2025-08-05 to 2025-09-04
INFO 2025-09-04 17:52:52,159 services 96922 6133706752 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/BAJAJFINSV_minute_20250805_to_20250904_20250904_175252.json
INFO 2025-09-04 17:53:04,866 strategy_service 96922 6133706752 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/BAJAJFINSV_minute_20250805_to_20250904_20250904_175252.json
INFO 2025-09-04 17:53:05,028 strategy_service 96922 6133706752 Saved 29 signals for BAJAJFINSV
INFO 2025-09-04 17:53:05,030 strategy_service 96922 6133706752 Saved backtest results for BAJAJFINSV
INFO 2025-09-04 17:53:11,765 views 96922 6133706752 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/BAJAJFINSV_minute_20250805_to_20250904_20250904_175252.json
INFO 2025-09-04 17:53:11,770 views 96922 6133706752 Loaded file with 7875 records
INFO 2025-09-04 17:53:11,772 views 96922 6133706752 Extracted 7875 valid OHLC records
INFO 2025-09-04 17:53:56,209 services 96922 6133706752 Fetching data for token 256265 from 2025-08-05 to 2025-09-04
INFO 2025-09-04 17:53:56,596 services 96922 6133706752 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/NIFTY50_minute_20250805_to_20250904_20250904_175356.json
ERROR 2025-09-04 18:12:20,051 views 97593 6170619904 Error exporting backtest to Excel: Excel does not support datetimes with timezones. Please ensure that datetimes are timezone unaware before writing to Excel.
Traceback (most recent call last):
  File "/Users/prathameshkale/Documents/django-automation/stock_data/views.py", line 936, in export_backtest_to_excel
    signals_df.to_excel(writer, sheet_name='Trade_Signals', index=False)
    ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/pandas/util/_decorators.py", line 333, in wrapper
    return func(*args, **kwargs)
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/pandas/core/generic.py", line 2436, in to_excel
    formatter.write(
    ~~~~~~~~~~~~~~~^
        excel_writer,
        ^^^^^^^^^^^^^
    ...<6 lines>...
        engine_kwargs=engine_kwargs,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/pandas/io/formats/excel.py", line 952, in write
    writer._write_cells(
    ~~~~~~~~~~~~~~~~~~~^
        formatted_cells,
        ^^^^^^^^^^^^^^^^
    ...<3 lines>...
        freeze_panes=freeze_panes,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/pandas/io/excel/_openpyxl.py", line 486, in _write_cells
    for cell in cells:
                ^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/pandas/io/formats/excel.py", line 890, in get_formatted_cells
    cell.val = self._format_value(cell.val)
               ~~~~~~~~~~~~~~~~~~^^^^^^^^^^
  File "/Users/prathameshkale/Documents/django-automation/venv/lib/python3.13/site-packages/pandas/io/formats/excel.py", line 607, in _format_value
    raise ValueError(
    ...<3 lines>...
    )
ValueError: Excel does not support datetimes with timezones. Please ensure that datetimes are timezone unaware before writing to Excel.
INFO 2025-09-04 18:17:42,247 views 683 6129692672 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/HINDALCO_minute_20250804_to_20250903_20250903_151423.json
INFO 2025-09-04 18:17:42,251 views 683 6129692672 Loaded file with 7875 records
INFO 2025-09-04 18:17:42,270 views 683 6129692672 Extracted 7875 valid OHLC records
INFO 2025-09-04 18:18:14,377 views 683 6129692672 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/HINDALCO_minute_20250804_to_20250903_20250903_151423.json
INFO 2025-09-04 18:18:14,381 views 683 6129692672 Loaded file with 7875 records
INFO 2025-09-04 18:18:14,384 views 683 6129692672 Extracted 7875 valid OHLC records
INFO 2025-09-04 18:18:26,549 strategy_service 683 6129692672 Running strategy on file: /Users/prathameshkale/Documents/django-automation/data_storage/NIFTY50_minute_20250805_to_20250904_20250904_175356.json
INFO 2025-09-04 18:18:26,706 strategy_service 683 6129692672 Saved 22 signals for NIFTY50
INFO 2025-09-04 18:18:26,707 strategy_service 683 6129692672 Saved backtest results for NIFTY50
INFO 2025-09-04 18:18:33,871 views 683 6129692672 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/NIFTY50_minute_20250805_to_20250904_20250904_175356.json
INFO 2025-09-04 18:18:33,876 views 683 6129692672 Loaded file with 7875 records
INFO 2025-09-04 18:18:33,878 views 683 6129692672 Extracted 7875 valid OHLC records
INFO 2025-09-04 18:19:13,527 views 683 6129692672 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/NIFTY50_minute_20250805_to_20250904_20250904_175356.json
INFO 2025-09-04 18:19:13,532 views 683 6129692672 Loaded file with 7875 records
INFO 2025-09-04 18:19:13,536 views 683 6129692672 Extracted 7875 valid OHLC records
INFO 2025-09-04 18:20:10,077 views 683 6129692672 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/BAJAJFINSV_minute_20250805_to_20250904_20250904_175252.json
INFO 2025-09-04 18:20:10,081 views 683 6129692672 Loaded file with 7875 records
INFO 2025-09-04 18:20:10,083 views 683 6129692672 Extracted 7875 valid OHLC records
INFO 2025-09-04 18:32:23,496 views 683 6129692672 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/NIFTY50_minute_20250805_to_20250904_20250904_175356.json
INFO 2025-09-04 18:32:23,501 views 683 6129692672 Loaded file with 7875 records
INFO 2025-09-04 18:32:23,503 views 683 6129692672 Extracted 7875 valid OHLC records
INFO 2025-09-04 18:33:14,012 views 683 6129692672 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/BAJAJFINSV_minute_20250805_to_20250904_20250904_175252.json
INFO 2025-09-04 18:33:14,017 views 683 6129692672 Loaded file with 7875 records
INFO 2025-09-04 18:33:14,019 views 683 6129692672 Extracted 7875 valid OHLC records
INFO 2025-09-04 18:33:21,371 views 683 6129692672 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/NIFTY50_minute_20250805_to_20250904_20250904_175356.json
INFO 2025-09-04 18:33:21,375 views 683 6129692672 Loaded file with 7875 records
INFO 2025-09-04 18:33:21,378 views 683 6129692672 Extracted 7875 valid OHLC records
INFO 2025-09-04 18:35:57,345 views 683 6129692672 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/BAJAJFINSV_minute_20250805_to_20250904_20250904_175252.json
INFO 2025-09-04 18:35:57,349 views 683 6129692672 Loaded file with 7875 records
INFO 2025-09-04 18:35:57,351 views 683 6129692672 Extracted 7875 valid OHLC records
INFO 2025-09-05 05:58:01,823 services 68607 6199095296 Fetching data for token 1195265 from 2025-08-06 to 2025-09-05
ERROR 2025-09-05 05:58:01,920 services 68607 6199095296 Error fetching historical data: Incorrect `api_key` or `access_token`.
INFO 2025-09-05 05:58:01,923 services 68607 6199095296 Generated 1000 sample records
INFO 2025-09-05 05:58:01,934 services 68607 6199095296 Data saved to /Users/prathameshkale/Documents/django-automation/data_storage/BANKBEES_minute_20250806_to_20250905_20250905_055801.json
INFO 2025-09-05 16:44:20,585 views 97739 6195359744 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/BAJAJFINSV_minute_20250805_to_20250904_20250904_175252.json
INFO 2025-09-05 16:44:20,590 views 97739 6195359744 Loaded file with 7875 records
INFO 2025-09-05 16:44:20,593 views 97739 6195359744 Extracted 7875 valid OHLC records
INFO 2025-09-06 05:04:58,307 views 2813 6189821952 Found matching symbol file: /Users/prathameshkale/Documents/django-automation/data_storage/NIFTY50_minute_20250805_to_20250904_20250904_175356.json
INFO 2025-09-06 05:04:58,311 views 2813 6189821952 Loaded file with 7875 records
INFO 2025-09-06 05:04:58,314 views 2813 6189821952 Extracted 7875 valid OHLC records
